src/prediqt.ts

```diff
@@ -84,35 +84,17 @@ export class Prediqt {
     }
 
     /**
-     * Cancel an order of type No
-     */
-    public async cancelOrderNo(user: string, marketId: number, id: number): Promise<any> {
-        return await this.api.transact(
-            {
-                actions: [{
-                    account: this.contractName,
-                    name: "cnclorderno",
-                    authorization: this.auth,
-                    data: {
-                        user,
-                        market_id: marketId,
-                        id,
-                    },
-                }],
-            },
-            this.transactParams,
-        );
-    }
-
-    /**
-     * Cancel an order of type Yes
+     * Cancel an order
      */
-    public async cancelOrderYes(user: string, marketId: number, id: number): Promise<any> {
+    public async cancelOrder(nameId: string, user: string, marketId: number, id: number): Promise<any> {
```

Comment:
we can use https://www.typescriptlang.org/docs/handbook/enums.html

Reply:
I will improve during my work with the lib